src/main/java/io/appium/java_client/pagefactory/AppiumFieldDecorator.java

```diff
@@ -62,7 +62,7 @@
     private static final List<Class<? extends WebElement>> availableElementClasses = ImmutableList.of(WebElement.class,
             RemoteWebElement.class, MobileElement.class, AndroidElement.class,
             IOSElement.class, WindowsElement.class);
-    public static long DEFAULT_IMPLICITLY_WAIT_TIMEOUT = 1;
+    public static long DEFAULT_TIMEOUT = 1;
```

Comment:
we can deprecate the constant of TimeUnit type if we use Duration type for DEFAULT_TIMEOUT